.dockerignore

```diff
@@ -2,3 +2,5 @@
 .git
 website
 assets
+private
+e2e
```

Comment:
do we need .env here just in case? or we never never push to Docker from local?

Reply:
Ok!